### ISSUE TITLE
want keras to work on GPU, but actually on CPU

### ISSUE THREAD

**Issue by XingyuGuUCSD** [tensorflow/tensorflow#16428]
System information:
OS Platform and Distribution: Linux Ubuntu 16.04:
TensorFlow installed from anaconda:
Keras installed from pip
TensorFlow version 1.4.1
Bazel version : None:
CUDA/cuDNN version : CUDA V7.5.17, CuDNN v6.0:
GPU: GeForce GeForce GTX 1050 Ti(3.94GB):
Code: [mnist_mlp.py](https://github.com/antoniosehk/keras-tensorflow-windows-installation/blob/master/examples/mnist_mlp.py)

When run the code, it shows the following log:

> Using TensorFlow backend.
2018-01-25 17:31:16.572013: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-01-25 17:31:16.679268: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-01-25 17:31:16.679499: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 0 with properties:
name: GeForce GTX 1050 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.468
pciBusID: 0000:01:00.0
totalMemory: 3.94GiB freeMemory: 3.46GiB
2018-01-25 17:31:16.679513: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1050 Ti, pci bus id: 0000:01:00.0, compute capability: 6.1)

Have no idea why it shows: 

> 2018-01-25 17:31:16.679268: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero

And the code is actually running on the CPU, although the log shows the info of GPU

What have tried: stackoverflow :)

**Comment by tatianashp** [comment]
I apologize, but I do not understand your question. Do you see an error when executing your program?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tatianashp** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
